src/main/resources/openapi/schemas/reportUseOverTime.json

```diff
@@ -0,0 +1,51 @@
+{
+  "description": "report data entry (agreement line information)",
+  "type": "object",
+  "properties": {
+    "agreementId": {
+      "type": "string",
+      "description": "agreement identifier (UUID)"
+    },
+    "accessCountTimeSlots": {
+      "description": "List of time slots, for example [\"2020-03\", \"2020-04\", \"2020-05\"]",
```

Comment:
Rephrase to "List of reporting periods, for example ..."

Reply:
Rephrased.